build/template-Build-run-tests-sign.yml

```diff
@@ -136,8 +145,6 @@ steps:
     verbosityPack: 'Minimal'
     configuration: $(BuildConfiguration)
     packagesToPack: '$(WilsonSourceDirectory)Product.proj'
-    feedsToUse: 'config'
-    nugetConfigPath: Nuget.config
     externalFeedCredentials: 'Internal Analyzers'
```

Comment:
In a different PR, you can also remove this one, and remove the service connection from the NuGetAuthenticate@1 tasks

Reply:
Will do🫡